src/rsz/src/Resizer.tcl

```diff
@@ -91,6 +91,38 @@ proc set_layer_rc { args } {
   }
 }
 
+sta::define_cmd_args "report_layer_rc" {[-corner corner]}
+proc report_layer_rc { args } {
+  sta::parse_key_args "report_layer_rc" args \
+    keys {-corner} \
+    flags {}
+  set corner [sta::parse_corner_or_all keys]
+  set tech [ord::get_db_tech]
+  set no_routing_layers [$tech getRoutingLayerCount]
+  ord::ensure_units_initialized
+  set res_unit "[sta::unit_scaled_suffix "resistance"]/[sta::unit_scaled_suffix "distance"]"
+  set cap_unit "[sta::unit_scaled_suffix "capacitance"]/[sta::unit_scaled_suffix "distance"]"
+  set res_convert [expr [sta::resistance_sta_ui 1.0] / [sta::distance_sta_ui 1.0]]
+  set cap_convert [expr [sta::capacitance_sta_ui 1.0] / [sta::distance_sta_ui 1.0]]
+
+  puts "   Layer   | Unit Resistance | Unit Capacitance "
+  puts [format "           | %15s | %16s" [format "(%s)" $res_unit] [format "(%s)" $cap_unit]]
+  puts "------------------------------------------------"
+  for { set i 1 } { $i <= $no_routing_layers } { incr i } {
+    set layer [$tech findRoutingLayer $i]
```

Comment:
you should also report for the via layers

Reply:
It can be added to `report_layer_rc` but they need a separate table since the units are different